src/Agent.php

```diff
@@ -39,6 +39,7 @@ class Agent extends Mobile_Detect {
     protected static $additionalBrowsers = array(
         'Opera'             => 'Opera|OPR',
         'Edge'              => 'Edge',
+        'Vivaldi'           => 'Vivaldi', //Must be above Chrome, otherwise it'll fail
```

Comment:
Alright move this comment above the variable in https://github.com/jenssegers/agent/pull/60/commits/4e0a0a4d8fc20e28d22f9584dc9a01fc51ddf90e